gokart/task.py

```diff
@@ -37,7 +37,7 @@ class TaskOnKart(luigi.Task):
     modification_time_check = luigi.BoolParameter(
         default=False,
         description='If this is true, this task will not run only if all input and output files exist,'
-        ' and all input files are modified before output file are modified.',
+                    ' and all input files are modified before output file are modified.',
```

Comment:
[nits] 

`' and all input` should be `'and all input` .
(heading white space is unnecessary)